src/components/homepage/HowDoesItWork/HowDoesItWork.tsx

```diff
@@ -1,9 +1,84 @@
-import { Heading2 } from '../../Headings/Heading';
+import React from 'react';
+
 import { HomepageSection } from '../HomepageSection/HomepageSection';
+import step1 from '../../../assets/images/howItWorks/step1.png';
+import step2 from '../../../assets/images/howItWorks/step2.png';
+import step3 from '../../../assets/images/howItWorks/step3.png';
+import step4 from '../../../assets/images/howItWorks/step4.png';
+import step5 from '../../../assets/images/howItWorks/step5.png';
 import style from './HowDoesItWork.module.scss';
 
-export const HowDoesItWork = () => (
-  <HomepageSection className={style.section}>
-    <Heading2>how does it work?</Heading2>
+type Step = {
+  number: string;
+  title: string;
+  description: string;
+};
+
+const steps: (Step & { image: string })[] = [
+  {
+    number: '01',
+    image: step1,
+    title: 'Join a club or start your own',
+    description:
+      'Find a club that interests you or create a new one and invite members.',
+  },
+  {
+    number: '02',
+    image: step2,
+    title: 'Suggest a book',
+    description:
+      "Each member can suggest a book they'd like to read. One book is randomly selected from the suggestions.",
+  },
+  {
+    number: '03',
+    image: step3,
+    title: 'Read the selected book',
+    description:
+      "Read the chosen book within the given timeframe so you're ready for the discussion.",
+  },
+  {
+    number: '04',
+    image: step4,
+    title: 'Meet and share your thoughts',
+    description:
+      'Gather with your club to exchange opinions, ideas, and impressions about the book.',
+  },
+  {
+    number: '05',
+    image: step5,
+    title: 'Rate the book',
+    description: 'Write a review to inspire other members and readers.',
+  },
+];
+
+const StepCard: React.FC<Step> = ({ number, title, description }) => (
```

Comment:
StepCardProps? Move it closer to the component

Reply:
Good feeling for NOT creating another component (another file) when the file is this small. Just be careful, this should be used sparely and on really small files (I would say max 100 LOC)